javascript/javascript3/week3/promise.js

```diff
@@ -0,0 +1,33 @@
+//Getting into promises
+//Select 3 classmates github username that you want to show repositories for.
+const username1 = "sandhana14";
+const username2 = "ccshah298";
+const username3 = "JyothiNandyala";
+//Fetch all the 3 classmates repositories
+const promise1 = fetch(
+  `https://api.github.com/search/repositories?q=user:${username1}`
```

Comment:
use a separate const for `https://api.github.com/search/repositories?q=user:` Or extract it into a function that takes a username as a parameter.